src/transaction/log_append.cpp

```diff
@@ -1008,6 +1008,7 @@ prior_lsa_gen_undoredo_record_from_crumbs (THREAD_ENTRY *thread_p, LOG_PRIOR_NOD
 	    {
 	      assert (tdes->mvccinfo.sub_ids.size () == 1);
 	      *mvccid_p = tdes->mvccinfo.sub_ids.back ();
+	      assert (MVCCID_IS_VALID (tdes->mvccinfo.id));
```

Comment:
Although I found other instances where we check IS_VALUE on mvcc_id from tdes, I expect this should only contain "NORMAL".
MVCCID_ALL_VISIBLE should be used only as special value on records (data/log), or as return value from visibility functions, not as mvccid in a transaction.